### ISSUE TITLE
articles.json not copied by install

### ISSUE THREAD

**Issue by rafwiewiora** [bowman-lab/enspara#187]
Had to manually copy articles.json after installation, otherwise:
```
Traceback (most recent call last):
  File "/home/rafal.wiewiora/repos/enspara/enspara/apps/collect_cards.py", line 33, in <module>
    from enspara.cards import cards
  File "/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/cards/__init__.py", line 4, in <module>
    from .cards import *
  File "/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/cards/cards.py", line 6, in <module>
    from ..info_theory import mutual_info
  File "/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/info_theory/__init__.py", line 6, in <module>
    from . import exposons
  File "/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/info_theory/exposons.py", line 8, in <module>
    from enspara.citation import cite
  File "/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/citation/__init__.py", line 3, in <module>
    from . import citation
  File "/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/citation/citation.py", line 50, in <module>
    CITATION_DB = load_citation_db()
  File "/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/citation/citation.py", line 10, in load_citation_db
    with open(os.path.join(os.path.dirname(__file__), 'articles.json')) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/rafal.wiewiora/anaconda3/lib/python3.6/site-packages/enspara-0.1.0-py3.6-linux-x86_64.egg/enspara/citation/articles.json'
```

**Comment by justinrporter** [comment]
Awesome catch--thanks! I probably merged this a bit before it was ready... 😒

**Comment by justinrporter** [comment]
Fixed in #188
